config/applications/api-filing-m2m-client.json.tpl

```diff
@@ -0,0 +1,281 @@
+{
+	"_id": "ApiFilingM2MClient",
+	"coreOAuth2ClientConfig": {
+		"userpassword": "ApiFilingM2MClient",
+		"loopbackInterfaceRedirection": {
+			"inherited": false,
+			"value": false
+		},
+		"defaultScopes": {
+			"inherited": false,
+			"value": []
+		},
+		"refreshTokenLifetime": {
+			"inherited": false,
+			"value": 604800
+		},
+		"scopes": {
+			"inherited": false,
+			"value": [
+                "openid",
+				"profile",
+				"email",
+                "https://account.companieshouse.gov.uk/user/profile.read",
+				"https://identity.company-information.service.gov.uk/user/profile.read",
+				"https://api.companieshouse.gov.uk/company/registered-office-address.update",
+				"https://api.company-information.service.gov.uk/company/registered-office-address.update",
+				"https://api.companieshouse.gov.uk/company/admin.write-full",
+				"https://api.company-information.service.gov.uk/company/admin.write-full"
```

Comment:
Will these need to be different in each in environment?

Reply:
I have no idea...